fix(curriculum): delineate span elements in typography (#55348)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/14-responsive-web-design-22/learn-typography-by-building-a-nutrition-label/615f7ecb09de9a938ef94756.md

---
id: 615f7ecb09de9a938ef94756
title: Step 56
challengeType: 0
dashedName: step-56
---

# --description--

> Below your last `p` element, add another `p` element with the text `Total Carbohydrate 37g 13%`. Like before, use `span` elements to make the text `Total Carbohydrate` and `13%` bold. 
> 
> Also add an additional `span` element around the `Total Carbohydrate 37g` text to have it aligned to the left and `13%` to the right.  Here you will nest a `span` element within a `span` element.

Reviewer: ```suggestion
Below your last `p` element, add another `p` element with the text `Total Carbohydrate 37g 13%`. Like before, use `span` elements to make the text `Total Carbohydrate` and `13%` bold. Then, wrap the nutrient and amount in a `span` for alignment again.
```
